coordinator/Dockerfile

```diff
@@ -1,20 +1,38 @@
-FROM openjdk:22-ea-17-slim-bookworm
+# BUILDER image
+FROM eclipse-temurin:21-jdk-noble AS builder
```

Comment:
@jonesho IIRC from our last our meeting with @julien-marchand we agreed on not having a builder image because the step commands bellow in `RUN` can be executed in the host machine.?

What is the motivation and value of having a builder image?

Reply:
@jpnovais Yes, but as mentioned above, the `jar` command was not available in JRE image, and hence we need to use JDK image here to use the `jar` command to extract and recreate the JAR archive based on the machine type (from `uname -m`). I was thinking about moving this "extract and recreate JAR" step to pipeline job but then we'd need separate "extract + build" jobs for `arm64` and `amd64` image respectively, which would take up more pipeline resources and runtime imo, and complicate the build process. Welcome for any better approach :)